*** testsBoth/A-12-06-M-ok.out	2022-10-30 22:57:52.925297000 +0000
--- testsBoth/A-12-06-M-ok.outhyp	2022-11-04 11:45:02.386310135 +0000
***************
*** 76,82 ****
  Escolha uma opção: 9
  Número do terminal: 969005
  Tipo de comunicação a iniciar (VOICE, VIDEO): VOICE
! O terminal de destino ('969005') está ocupado.
  OPERAÇÕES DO TERMINAL
  1 - Ligar
  2 - Colocar no silêncio
--- 76,82 ----
  Escolha uma opção: 9
  Número do terminal: 969005
  Tipo de comunicação a iniciar (VOICE, VIDEO): VOICE
! Iniciar comunicação: Operação inválida: O terminal de destino ('969005') está desligado.
  OPERAÇÕES DO TERMINAL
  1 - Ligar
  2 - Colocar no silêncio
***************
*** 85,92 ****
  5 - Retirar amigo
  6 - Pagamento
  7 - Consultar saldo
- 8 - Enviar comunicação de texto
- 9 - Iniciar comunicação
  11 - Mostrar comunicação em curso
  0 - Sair
  Escolha uma opção: 11
--- 85,90 ----
***************
*** 99,106 ****
  5 - Retirar amigo
  6 - Pagamento
  7 - Consultar saldo
- 8 - Enviar comunicação de texto
- 9 - Iniciar comunicação
  11 - Mostrar comunicação em curso
  0 - Sair
  Escolha uma opção: 0
--- 97,102 ----
