*** testsBoth/A-12-20-M-ok.out	2022-11-06 22:59:06.926927000 +0000
--- testsBoth/A-12-20-M-ok.outhyp	2022-11-12 20:18:34.328721134 +0000
***************
*** 118,124 ****
  11 - Mostrar comunicação em curso
  0 - Sair
  Escolha uma opção: 11
! VOICE|2|969001|969005|0|0|ONGOING
  OPERAÇÕES DO TERMINAL
  1 - Ligar
  2 - Colocar no silêncio
--- 118,124 ----
  11 - Mostrar comunicação em curso
  0 - Sair
  Escolha uma opção: 11
! VOICE|3|969001|969005|0|0|ONGOING
  OPERAÇÕES DO TERMINAL
  1 - Ligar
  2 - Colocar no silêncio
