*** testsBoth/A-12-04-M-ok.out	2022-10-30 22:57:52.731417000 +0000
--- testsBoth/A-12-04-M-ok.outhyp	2022-11-04 10:00:51.210184738 +0000
***************
*** 29,35 ****
  Escolha uma opção: 9
  Número do terminal: 969006
  Tipo de comunicação a iniciar (VOICE, VIDEO): VOICE
- O terminal de destino ('969006') está desligado.
  OPERAÇÕES DO TERMINAL
  1 - Ligar
  2 - Colocar no silêncio
--- 29,34 ----
***************
*** 38,45 ****
  5 - Retirar amigo
  6 - Pagamento
  7 - Consultar saldo
! 8 - Enviar comunicação de texto
! 9 - Iniciar comunicação
  11 - Mostrar comunicação em curso
  0 - Sair
  Escolha uma opção: 0
--- 37,43 ----
  5 - Retirar amigo
  6 - Pagamento
  7 - Consultar saldo
! 10 - Terminar comunicação activa
  11 - Mostrar comunicação em curso
  0 - Sair
  Escolha uma opção: 0
