*** testsBoth/A-14-02-M-ok.out	2022-10-30 22:57:54.301788000 +0000
--- testsBoth/A-14-02-M-ok.outhyp	2022-11-03 15:18:48.204119596 +0000
***************
*** 41,47 ****
  11 - Mostrar comunicação em curso
  0 - Sair
  Escolha uma opção: 11
- VOICE|1|969001|969011|0|0|ONGOING
  OPERAÇÕES DO TERMINAL
  1 - Ligar
  2 - Colocar no silêncio
--- 41,46 ----
***************
*** 64,74 ****
  BASIC|969003|cli001|OFF|0|0
  BASIC|969006|cli002|IDLE|0|0
  BASIC|969007|cli003|IDLE|0|0
! BASIC|969008|cli003|SILENCE|0|0
! FANCY|969011|cli001|BUSY|0|0
! FANCY|969013|cli001|SILENCE|0|0
! FANCY|969014|cli002|IDLE|0|0
! FANCY|969015|cli003|IDLE|0|0
  GESTÃO DE TERMINAIS
  1 - Mostrar todos os terminais
  2 - Registar terminal
--- 63,73 ----
  BASIC|969003|cli001|OFF|0|0
  BASIC|969006|cli002|IDLE|0|0
  BASIC|969007|cli003|IDLE|0|0
! BASIC|969008|cli003|SILENSE|0|0
! BASIC|969011|cli001|IDLE|0|0
! BASIC|969013|cli001|SILENSE|0|0
! BASIC|969014|cli002|IDLE|0|0
! BASIC|969015|cli003|IDLE|0|0
  GESTÃO DE TERMINAIS
  1 - Mostrar todos os terminais
  2 - Registar terminal
