*** testsBoth/A-15-01-M-ok.out	2022-10-30 22:57:54.715456000 +0000
--- testsBoth/A-15-01-M-ok.outhyp	2022-11-04 11:45:07.422153713 +0000
***************
*** 60,66 ****
  CONSULTAS
  1 - Mostrar todas as comunicações
  2 - Mostrar comunicações feitas por um cliente
! 3 - Mostrar comunicações recebidas por um cliente
  4 - Mostrar clientes sem dívidas
  5 - Mostrar clientes com dívidas
  6 - Mostrar terminais sem actividade
--- 60,66 ----
  CONSULTAS
  1 - Mostrar todas as comunicações
  2 - Mostrar comunicações feitas por um cliente
! 3 - Mostrar comunicações feitas por um cliente
  4 - Mostrar clientes sem dívidas
  5 - Mostrar clientes com dívidas
  6 - Mostrar terminais sem actividade
***************
*** 71,77 ****
  CONSULTAS
  1 - Mostrar todas as comunicações
  2 - Mostrar comunicações feitas por um cliente
! 3 - Mostrar comunicações recebidas por um cliente
  4 - Mostrar clientes sem dívidas
  5 - Mostrar clientes com dívidas
  6 - Mostrar terminais sem actividade
--- 71,77 ----
  CONSULTAS
  1 - Mostrar todas as comunicações
  2 - Mostrar comunicações feitas por um cliente
! 3 - Mostrar comunicações feitas por um cliente
  4 - Mostrar clientes sem dívidas
  5 - Mostrar clientes com dívidas
  6 - Mostrar terminais sem actividade
***************
*** 140,146 ****
  CONSULTAS
  1 - Mostrar todas as comunicações
  2 - Mostrar comunicações feitas por um cliente
! 3 - Mostrar comunicações recebidas por um cliente
  4 - Mostrar clientes sem dívidas
  5 - Mostrar clientes com dívidas
  6 - Mostrar terminais sem actividade
--- 140,146 ----
  CONSULTAS
  1 - Mostrar todas as comunicações
  2 - Mostrar comunicações feitas por um cliente
! 3 - Mostrar comunicações feitas por um cliente
  4 - Mostrar clientes sem dívidas
  5 - Mostrar clientes com dívidas
  6 - Mostrar terminais sem actividade
***************
*** 152,158 ****
  CONSULTAS
  1 - Mostrar todas as comunicações
  2 - Mostrar comunicações feitas por um cliente
! 3 - Mostrar comunicações recebidas por um cliente
  4 - Mostrar clientes sem dívidas
  5 - Mostrar clientes com dívidas
  6 - Mostrar terminais sem actividade
--- 152,158 ----
  CONSULTAS
  1 - Mostrar todas as comunicações
  2 - Mostrar comunicações feitas por um cliente
! 3 - Mostrar comunicações feitas por um cliente
  4 - Mostrar clientes sem dívidas
  5 - Mostrar clientes com dívidas
  6 - Mostrar terminais sem actividade
