*** testsBoth/A-17-04-M-ok.out	2022-10-30 22:57:56.125970000 +0000
--- testsBoth/A-17-04-M-ok.outhyp	2022-11-04 10:01:03.153819621 +0000
***************
*** 28,33 ****
--- 28,35 ----
  0 - Sair
  Escolha uma opção: 4
  Número do terminal: 969001
+ Número do terminal: 4
+ Adicionar amigo: Operação inválida: O terminal '4' não existe.
  OPERAÇÕES DO TERMINAL
  1 - Ligar
  2 - Colocar no silêncio
***************
*** 40,47 ****
  9 - Iniciar comunicação
  11 - Mostrar comunicação em curso
  0 - Sair
! Escolha uma opção: 4
! Número do terminal: 969008
  OPERAÇÕES DO TERMINAL
  1 - Ligar
  2 - Colocar no silêncio
--- 42,49 ----
  9 - Iniciar comunicação
  11 - Mostrar comunicação em curso
  0 - Sair
! Escolha uma opção: 969008
! Opção inválida!
  OPERAÇÕES DO TERMINAL
  1 - Ligar
  2 - Colocar no silêncio
***************
*** 56,61 ****
--- 58,65 ----
  0 - Sair
  Escolha uma opção: 4
  Número do terminal: 969007
+ Número do terminal: 0
+ Adicionar amigo: Operação inválida: O terminal '0' não existe.
  OPERAÇÕES DO TERMINAL
  1 - Ligar
  2 - Colocar no silêncio
***************
*** 68,96 ****
  9 - Iniciar comunicação
  11 - Mostrar comunicação em curso
  0 - Sair
- Escolha uma opção: 0
- GESTÃO DE TERMINAIS
- 1 - Mostrar todos os terminais
- 2 - Registar terminal
- 3 - Operações de terminal
- 0 - Sair
  Escolha uma opção: 1
! BASIC|969001|cli001|IDLE|0|0
! BASIC|969003|cli001|OFF|0|0
! BASIC|969006|cli002|IDLE|0|0|969001
! BASIC|969007|cli003|IDLE|0|0
! BASIC|969008|cli003|SILENCE|0|0
! FANCY|969011|cli001|IDLE|0|0|969001,969007,969008,969014
! FANCY|969013|cli001|SILENCE|0|0
! FANCY|969014|cli002|IDLE|0|0
! FANCY|969015|cli003|IDLE|0|0
! GESTÃO DE TERMINAIS
! 1 - Mostrar todos os terminais
! 2 - Registar terminal
! 3 - Operações de terminal
  0 - Sair
! Escolha uma opção: 3
! Número do terminal: 969011
  OPERAÇÕES DO TERMINAL
  1 - Ligar
  2 - Colocar no silêncio
--- 72,94 ----
  9 - Iniciar comunicação
  11 - Mostrar comunicação em curso
  0 - Sair
  Escolha uma opção: 1
! Número do terminal: 3
! Ligar: Operação inválida: O terminal '3' não existe.
! OPERAÇÕES DO TERMINAL
! 1 - Ligar
! 2 - Colocar no silêncio
! 3 - Desligar
! 4 - Adicionar amigo
! 5 - Retirar amigo
! 6 - Pagamento
! 7 - Consultar saldo
! 8 - Enviar comunicação de texto
! 9 - Iniciar comunicação
! 11 - Mostrar comunicação em curso
  0 - Sair
! Escolha uma opção: 969011
! Opção inválida!
  OPERAÇÕES DO TERMINAL
  1 - Ligar
  2 - Colocar no silêncio
***************
*** 105,110 ****
--- 103,110 ----
  0 - Sair
  Escolha uma opção: 5
  Número do terminal: 969007
+ Número do terminal: 5
+ Retirar amigo: Operação inválida: O terminal '5' não existe.
  OPERAÇÕES DO TERMINAL
  1 - Ligar
  2 - Colocar no silêncio
***************
*** 117,124 ****
  9 - Iniciar comunicação
  11 - Mostrar comunicação em curso
  0 - Sair
! Escolha uma opção: 5
! Número do terminal: 969014
  OPERAÇÕES DO TERMINAL
  1 - Ligar
  2 - Colocar no silêncio
--- 117,124 ----
  9 - Iniciar comunicação
  11 - Mostrar comunicação em curso
  0 - Sair
! Escolha uma opção: 969014
! Opção inválida!
  OPERAÇÕES DO TERMINAL
  1 - Ligar
  2 - Colocar no silêncio
***************
*** 133,138 ****
--- 133,140 ----
  0 - Sair
  Escolha uma opção: 5
  Número do terminal: 969001
+ Número do terminal: 0
+ Retirar amigo: Operação inválida: O terminal '0' não existe.
  OPERAÇÕES DO TERMINAL
  1 - Ligar
  2 - Colocar no silêncio
***************
*** 145,173 ****
  9 - Iniciar comunicação
  11 - Mostrar comunicação em curso
  0 - Sair
- Escolha uma opção: 0
- GESTÃO DE TERMINAIS
- 1 - Mostrar todos os terminais
- 2 - Registar terminal
- 3 - Operações de terminal
- 0 - Sair
  Escolha uma opção: 1
! BASIC|969001|cli001|IDLE|0|0
! BASIC|969003|cli001|OFF|0|0
! BASIC|969006|cli002|IDLE|0|0|969001
! BASIC|969007|cli003|IDLE|0|0
! BASIC|969008|cli003|SILENCE|0|0
! FANCY|969011|cli001|IDLE|0|0|969008
! FANCY|969013|cli001|SILENCE|0|0
! FANCY|969014|cli002|IDLE|0|0
! FANCY|969015|cli003|IDLE|0|0
! GESTÃO DE TERMINAIS
! 1 - Mostrar todos os terminais
! 2 - Registar terminal
! 3 - Operações de terminal
  0 - Sair
! Escolha uma opção: 3
! Número do terminal: 969011
  OPERAÇÕES DO TERMINAL
  1 - Ligar
  2 - Colocar no silêncio
--- 147,169 ----
  9 - Iniciar comunicação
  11 - Mostrar comunicação em curso
  0 - Sair
  Escolha uma opção: 1
! Número do terminal: 3
! Ligar: Operação inválida: O terminal '3' não existe.
! OPERAÇÕES DO TERMINAL
! 1 - Ligar
! 2 - Colocar no silêncio
! 3 - Desligar
! 4 - Adicionar amigo
! 5 - Retirar amigo
! 6 - Pagamento
! 7 - Consultar saldo
! 8 - Enviar comunicação de texto
! 9 - Iniciar comunicação
! 11 - Mostrar comunicação em curso
  0 - Sair
! Escolha uma opção: 969011
! Opção inválida!
  OPERAÇÕES DO TERMINAL
  1 - Ligar
  2 - Colocar no silêncio
***************
*** 182,187 ****
--- 178,185 ----
  0 - Sair
  Escolha uma opção: 5
  Número do terminal: 969008
+ Número do terminal: 0
+ Retirar amigo: Operação inválida: O terminal '0' não existe.
  OPERAÇÕES DO TERMINAL
  1 - Ligar
  2 - Colocar no silêncio
***************
*** 194,221 ****
  9 - Iniciar comunicação
  11 - Mostrar comunicação em curso
  0 - Sair
- Escolha uma opção: 0
- GESTÃO DE TERMINAIS
- 1 - Mostrar todos os terminais
- 2 - Registar terminal
- 3 - Operações de terminal
- 0 - Sair
  Escolha uma opção: 1
! BASIC|969001|cli001|IDLE|0|0
! BASIC|969003|cli001|OFF|0|0
! BASIC|969006|cli002|IDLE|0|0|969001
! BASIC|969007|cli003|IDLE|0|0
! BASIC|969008|cli003|SILENCE|0|0
! FANCY|969011|cli001|IDLE|0|0
! FANCY|969013|cli001|SILENCE|0|0
! FANCY|969014|cli002|IDLE|0|0
! FANCY|969015|cli003|IDLE|0|0
  GESTÃO DE TERMINAIS
  1 - Mostrar todos os terminais
  2 - Registar terminal
  3 - Operações de terminal
  0 - Sair
! Escolha uma opção: 0
  MENU PRINCIPAL
  1 - Abrir
  2 - Guardar
--- 192,219 ----
  9 - Iniciar comunicação
  11 - Mostrar comunicação em curso
  0 - Sair
  Escolha uma opção: 1
! Número do terminal: 0
! Ligar: Operação inválida: O terminal '0' não existe.
! OPERAÇÕES DO TERMINAL
! 1 - Ligar
! 2 - Colocar no silêncio
! 3 - Desligar
! 4 - Adicionar amigo
! 5 - Retirar amigo
! 6 - Pagamento
! 7 - Consultar saldo
! 8 - Enviar comunicação de texto
! 9 - Iniciar comunicação
! 11 - Mostrar comunicação em curso
! 0 - Sair
! Escolha uma opção: 0
  GESTÃO DE TERMINAIS
  1 - Mostrar todos os terminais
  2 - Registar terminal
  3 - Operações de terminal
  0 - Sair
! Escolha uma opção: Fim de entrada de dados (R-EOF): pt.tecnico.uilib.text.RuntimeEOFException: Fim do fluxo de dados de entrada
  MENU PRINCIPAL
  1 - Abrir
  2 - Guardar
***************
*** 224,227 ****
  5 - Menu de consultas
  6 - Ver saldo global
  0 - Sair
! Escolha uma opção: 0
--- 222,225 ----
  5 - Menu de consultas
  6 - Ver saldo global
  0 - Sair
! Escolha uma opção: Fim de entrada de dados (R-EOF): pt.tecnico.uilib.text.RuntimeEOFException: Fim do fluxo de dados de entrada
