*** testsBoth/A-19-01-M-ok.out	2022-10-30 22:57:57.557118000 +0000
--- testsBoth/A-19-01-M-ok.outhyp	2022-11-03 15:18:58.355801445 +0000
***************
*** 17,27 ****
  BASIC|969003|cli001|OFF|0|0
  BASIC|969006|cli002|IDLE|0|0
  BASIC|969007|cli003|IDLE|0|0
! BASIC|969008|cli003|SILENCE|0|0
! FANCY|969011|cli001|IDLE|0|0
! FANCY|969013|cli001|SILENCE|0|0
! FANCY|969014|cli002|IDLE|0|0
! FANCY|969015|cli003|IDLE|0|0
  GESTÃO DE TERMINAIS
  1 - Mostrar todos os terminais
  2 - Registar terminal
--- 17,27 ----
  BASIC|969003|cli001|OFF|0|0
  BASIC|969006|cli002|IDLE|0|0
  BASIC|969007|cli003|IDLE|0|0
! BASIC|969008|cli003|SILENSE|0|0
! BASIC|969011|cli001|IDLE|0|0
! BASIC|969013|cli001|SILENSE|0|0
! BASIC|969014|cli002|IDLE|0|0
! BASIC|969015|cli003|IDLE|0|0
  GESTÃO DE TERMINAIS
  1 - Mostrar todos os terminais
  2 - Registar terminal
***************
*** 40,45 ****
--- 40,47 ----
  11 - Mostrar comunicação em curso
  0 - Sair
  Escolha uma opção: 1
+ Número do terminal: 0
+ Ligar: Operação inválida: O terminal '0' não existe.
  OPERAÇÕES DO TERMINAL
  1 - Ligar
  2 - Colocar no silêncio
***************
*** 48,79 ****
  5 - Retirar amigo
  6 - Pagamento
  7 - Consultar saldo
- 8 - Enviar comunicação de texto
- 9 - Iniciar comunicação
  11 - Mostrar comunicação em curso
  0 - Sair
- Escolha uma opção: 0
- GESTÃO DE TERMINAIS
- 1 - Mostrar todos os terminais
- 2 - Registar terminal
- 3 - Operações de terminal
- 0 - Sair
  Escolha uma opção: 1
! BASIC|969001|cli001|IDLE|0|0
! BASIC|969003|cli001|IDLE|0|0
! BASIC|969006|cli002|IDLE|0|0
! BASIC|969007|cli003|IDLE|0|0
! BASIC|969008|cli003|SILENCE|0|0
! FANCY|969011|cli001|IDLE|0|0
! FANCY|969013|cli001|SILENCE|0|0
! FANCY|969014|cli002|IDLE|0|0
! FANCY|969015|cli003|IDLE|0|0
  GESTÃO DE TERMINAIS
  1 - Mostrar todos os terminais
  2 - Registar terminal
  3 - Operações de terminal
  0 - Sair
! Escolha uma opção: 0
  MENU PRINCIPAL
  1 - Abrir
  2 - Guardar
--- 50,77 ----
  5 - Retirar amigo
  6 - Pagamento
  7 - Consultar saldo
  11 - Mostrar comunicação em curso
  0 - Sair
  Escolha uma opção: 1
! Número do terminal: 0
! Ligar: Operação inválida: O terminal '0' não existe.
! OPERAÇÕES DO TERMINAL
! 1 - Ligar
! 2 - Colocar no silêncio
! 3 - Desligar
! 4 - Adicionar amigo
! 5 - Retirar amigo
! 6 - Pagamento
! 7 - Consultar saldo
! 11 - Mostrar comunicação em curso
! 0 - Sair
! Escolha uma opção: 0
  GESTÃO DE TERMINAIS
  1 - Mostrar todos os terminais
  2 - Registar terminal
  3 - Operações de terminal
  0 - Sair
! Escolha uma opção: Fim de entrada de dados (R-EOF): pt.tecnico.uilib.text.RuntimeEOFException: Fim do fluxo de dados de entrada
  MENU PRINCIPAL
  1 - Abrir
  2 - Guardar
***************
*** 82,85 ****
  5 - Menu de consultas
  6 - Ver saldo global
  0 - Sair
! Escolha uma opção: 0
--- 80,83 ----
  5 - Menu de consultas
  6 - Ver saldo global
  0 - Sair
! Escolha uma opção: Fim de entrada de dados (R-EOF): pt.tecnico.uilib.text.RuntimeEOFException: Fim do fluxo de dados de entrada
