*** testsBoth/A-22-06-M-ok.out	2022-11-06 22:59:17.920538000 +0000
--- testsBoth/A-22-06-M-ok.outhyp	2022-11-12 20:19:31.834967061 +0000
***************
*** 195,204 ****
  0 - Sair
  Escolha uma opção: 3
  Identificador do cliente: cli001
- VOICE|1|969014|969001|1|20|FINISHED
- VOICE|2|969007|969011|1|20|FINISHED
- TEXT|3|969007|969011|6|10|FINISHED
- VOICE|4|969015|969013|1|20|FINISHED
  CONSULTAS
  1 - Mostrar todas as comunicações
  2 - Mostrar comunicações feitas por um cliente
--- 195,200 ----
