*** testsBoth/A-24-03-M-ok.out	2022-11-06 22:59:19.237546000 +0000
--- testsBoth/A-24-03-M-ok.outhyp	2022-11-13 17:18:47.291161961 +0000
***************
*** 142,147 ****
--- 142,148 ----
  7 - Mostrar terminais com saldo positivo
  0 - Sair
  Escolha uma opção: 4
+ CLIENT|cli003|Pedro Oliveira|103449|NORMAL|YES|1|40|20
  CLIENT|cli001|Manuel Pinheiro|103443|NORMAL|YES|1|0|0
  CLIENT|cli002|Pedro Pinheiro|103447|NORMAL|YES|1|0|0
  CONSULTAS
***************
*** 222,230 ****
  7 - Mostrar terminais com saldo positivo
  0 - Sair
  Escolha uma opção: 4
  CLIENT|cli001|Manuel Pinheiro|103443|NORMAL|YES|1|0|0
  CLIENT|cli002|Pedro Pinheiro|103447|NORMAL|YES|1|0|0
- CLIENT|cli003|Pedro Oliveira|103449|NORMAL|YES|1|60|0
  CONSULTAS
  1 - Mostrar todas as comunicações
  2 - Mostrar comunicações feitas por um cliente
--- 223,231 ----
  7 - Mostrar terminais com saldo positivo
  0 - Sair
  Escolha uma opção: 4
+ CLIENT|cli003|Pedro Oliveira|103449|NORMAL|YES|1|60|0
  CLIENT|cli001|Manuel Pinheiro|103443|NORMAL|YES|1|0|0
  CLIENT|cli002|Pedro Pinheiro|103447|NORMAL|YES|1|0|0
  CONSULTAS
  1 - Mostrar todas as comunicações
  2 - Mostrar comunicações feitas por um cliente
