*** testsBoth/A-24-05-M-ok.out	2022-11-06 22:59:19.423873000 +0000
--- testsBoth/A-24-05-M-ok.outhyp	2022-11-13 17:18:48.043139437 +0000
***************
*** 147,155 ****
  7 - Mostrar terminais com saldo positivo
  0 - Sair
  Escolha uma opção: 4
  CLIENT|cli001|Manuel Pinheiro|103443|NORMAL|YES|1|0|0
  CLIENT|cli002|Pedro Pinheiro|103447|NORMAL|YES|1|0|0
- CLIENT|cli003|Pedro Oliveira|103449|NORMAL|YES|1|10|0
  CONSULTAS
  1 - Mostrar todas as comunicações
  2 - Mostrar comunicações feitas por um cliente
--- 147,155 ----
  7 - Mostrar terminais com saldo positivo
  0 - Sair
  Escolha uma opção: 4
+ CLIENT|cli003|Pedro Oliveira|103449|NORMAL|YES|1|10|0
  CLIENT|cli001|Manuel Pinheiro|103443|NORMAL|YES|1|0|0
  CLIENT|cli002|Pedro Pinheiro|103447|NORMAL|YES|1|0|0
  CONSULTAS
  1 - Mostrar todas as comunicações
  2 - Mostrar comunicações feitas por um cliente
